### PR TITLE
[OSDOCS-6513]: Add platforms to hosted control planes overview

### PR DESCRIPTION
**Pull request by lahinson**
Version(s): 4.13+

Issue: https://issues.redhat.com/browse/OSDOCS-6513

Link to docs preview: https://61237--docspreview.netlify.app/openshift-enterprise/latest/hosted_control_planes/index.html

QE review:
- [x] QE has approved this change.

Additional information: This small update adds the platforms that hosted control planes is available on as a TP feature.

### CODE REVIEW

**Review by zanetworker — COMMENTED** [review]
Added a tiny nit pick, otherwise lgtm :)

File: modules/hosted-control-planes-overview.adoc [hunk 1]
```diff
@@ -10,7 +10,7 @@
 
 You can use hosted control planes for Red Hat {product-title} to reduce management costs, optimize cluster deployment time, and separate management and workload concerns so that you can focus on your applications.
 
-You can enable hosted control planes as a Technology Preview feature by using the link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.8/html/clusters/cluster_mce_overview#cluster_mce_overview[multicluster engine for Kubernetes operator version 2.0 or later] on Amazon Web Services (AWS).
+You can enable hosted control planes as a Technology Preview feature by using the link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.8/html/clusters/cluster_mce_overview#cluster_mce_overview[multicluster engine for Kubernetes operator version 2.0 or later] on Amazon Web Services (AWS), bare metal, or {VirtProductName}.
```
[the remark this answers is not in the document]
Comment (lahinson) [reply modules/hosted-control-planes-overview.adoc]: Fixed ✅

**Review by zanetworker — APPROVED** [review]
lgtm

**Comment by jiezhao16** [comment]
lgtm

**Comment by jab-rh** [comment]
@lahinson, LGTM, thanks!

**Pull request merged**